File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md

```diff
@@ -0,0 +1,207 @@
+                </div>
+                &nbsp;
+                <div class="rowdiv" style="border-width: 0.5px;border-style:solid; border-color: lightgray; padding: 1px 5px 7px 5px">
+                    Click the button to view the resultant PDF document being converted from Word document using DocIO. Please note that PDF viewer is required to view the resultant PDF.
+                    <div class="rowdiv" style="margin-top: 10px">
+                        <div class="celldiv">
+                            Select Document :
+                            @Html.TextBox("file", "", new { type = "file", accept = ".docx" }) <br />
+                        </div>
+                        <div class="rowdiv" style="margin-top: 8px">
+                            <input class="buttonStyle" type="submit" value="Convert to PDF" name="button" style="width:150px;height:27px" />
+                            <br />
+                            <div class="text-danger">
+                                @ViewBag.Message
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <br />
+      
+            </div>
+        </div>
+        Html.EndForm();
+    }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 6: Include the following namespaces in **HomeController.cs**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.Pdf;
+using Microsoft.AspNetCore.Hosting;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 9: Include the below code snippet in **HomeController.cs** for  **convert the Word document to Pdf**. 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+private Microsoft.AspNetCore.Hosting.IHostingEnvironment _env;
+public HomeController(Microsoft.AspNetCore.Hosting.IHostingEnvironment env)
+{
+    _env = env;
+}
+
+/// <summary>
+/// Convert Word document to PDF
+/// </summary>
+/// <param name="button"></param>
+/// <returns></returns>
+public IActionResult WordToPDF(string button)
+{
+
+    if (button == null)
+        return View("Index");
+
+    if (Request.Form.Files != null)
+    {
+        if (Request.Form.Files.Count == 0)
+        {
+            ViewBag.Message = string.Format("Browse a Word document and then click the button to convert as a PDF document");
+            return View("Index");
+        }
+        // Gets the extension from file.
+        string extension = Path.GetExtension(Request.Form.Files[0].FileName).ToLower();
+        // Compares extension with supported extensions.
+        if (extension == ".docx")
+        {
+            MemoryStream stream = new MemoryStream();
+            Request.Form.Files[0].CopyTo(stream);
+            try
+            {
+                //Open using Syncfusion
+                using (WordDocument document = new WordDocument(stream, Syncfusion.DocIO.FormatType.Docx))
```

Comment:
Why we need to mention full naming Syncfusion.DocIO.FormatType.Docx?

Reply:
Updated